homeassistant/components/london_underground/const.py

```diff
@@ -18,7 +18,6 @@
     "Elizabeth line",
     "Hammersmith & City",
     "Jubilee",
-    "London Overground",
```

Comment:
Instead of just removing it, let's first deprecate it and raise a proper issue for the user. Otherwise the validation of the integration will fail

Reply:
OK, somebody "deprecate it and raise a proper issue for the user".
I've no idea what that actually means needs doing.